app/src/main/res/drawable/buttonshape_calendarticket.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"     android:shape="rectangle" >
+    <corners
+        android:topLeftRadius="50dp"
+        android:topRightRadius="50dp"
+        android:bottomLeftRadius="50dp"
+        android:bottomRightRadius="50dp"
```

Comment:
Use "radius=..." if all edges are the same.

Reply:
delete this file because not find usage of it